surveys/2024-04/questions.md

```diff
@@ -0,0 +1,192 @@
+  - Country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
+
+- In what kind of organizations are you working in?
+  - big tech
+  - company
+  - startup
+  - university
+  - individual
+  - other
+
+- If working in a company, in which sector does your company operate? (single choice - Other should give an input)
+  - Energy
+  - Materials
+  - Industrials
+  - Health Care
+  - Financials
+  - Information Technology
+  - Communication Services
+  - Utilities
+  - Real Estate
+  - Other?
+
+## Node.js Usage
+
+- Which groups do you identify with? (multiple choice)
+  - Direct end users: Users who run tools themselves
+  - Application operators: Service and infrastructure providers
+  - Application developers: Frontend tools consumers, backend server authors, tools authors
+  - Library & package authors: Users who write libraries and packages to be included on other applications
+  - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+  - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+
+- Are you part of a group not covered ? If yes which one ? (open question)
+
+- What is your use cases of Node.js ? (multiple choice) (open question - Other should give an input)
+  - Cli tools
+  - Testing
+  - Development of APIs with Microservices
+  - Development of APIs with Serverless
+  - Development of APIs with Other
+  - Building static front end applications
+  - Deploying front end applications (Next.js, Remix, etc...)
+  - Script and automation (bots, scrapers)
+  - Proxy
+  - other
+
+## What Node.js binaries do you use
+
+- What is your Operating System for your local development environment ? (single choice - Other should give an input)
+  - Linux
+  - Windows
+  - macOS
+  - Linux with Docker
+  - Windows with Docker
+  - macOS with Docker
+  - Windows with WSL
+  - Other?
+
+- What is your Operating System in which you are running Node.js in production ? (multiple choice - Other should give an input)
+  - Linux/Unix
+  - Windows
+  - macOS
+  - Linux with Docker
+  - Windows with Docker
+  - macOS with Docker
+  - Other?
+
+- What architecture is the machine you are running Node.js in production? (multiple choice - Other should give an input)
+  - x64
+  - x32
+  - arm
+  - ppc
+  - s390
+  - Don't know
+  - Other?
+
+- How do you get your `node` executables? (multiple choice – Other should give an input)
+  - With a Node.js version manager: nvm, n, nave, nvs, volta, etc.
+  - Using the official installer
+  - Using package managers like npm or yarn and install Node.js as a package
+  - Using a system package manager: apt-get, brew, dnf, yum …
+  - Downloading binaries directly from Nodejs.org
+  - Building Node.js from Source
+  - Other
+
+- What package manager do you use ? (multiple choice - Other should give an input)
+  - npm
+  - yarn v1
+  - yarn v3
```

Comment:
```suggestion
  - yarn v4
```
or
```suggestion
  - Yarn Berry (2+)
```

Reply:
We can also use yarn classic and yarn modern
```suggestion
  - yarn modern (2+)
```